### PR TITLE
Add asdf-vm to Install of README

### PR DESCRIPTION
**Pull request by grimoh**
Add installation method with asdf-vm to Install of README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,16 @@ sudo dnf install k6   # use yum instead of dnf for older distros
...
 ```
 
+### asdf-vm
+
+If your kernel is Darwin or Linux, you can install with [asdf-vm](https://asdf-vm.com) by running:
+
+```
+asdf plugin-add k6
+asdf install k6 <K6_VERSION>
+asdf global <K&_VERSION>
```
Comment (imiric) [inline README.md]:
```suggestion
asdf global <K6_VERSION>
```

File: README.md [hunk 2]
```diff
@@ -93,6 +93,16 @@ sudo dnf install k6   # use yum instead of dnf for older distros
 docker pull loadimpact/k6
 ```
 
+### asdf-vm
+
+If your kernel is Darwin or Linux, you can install with [asdf-vm](https://asdf-vm.com) by running:
```
Comment (cuonglm) [inline README.md]: Maybe `On Linux/OSX,`

Comment (grimoh) [reply README.md]: Thank you.
Fixed.

File: README.md (hunk 1 above)
Comment (grimoh) [reply README.md]: Thank you.
Fixed.

**Review by cuonglm — DISMISSED** [review]

**Comment by grimoh** [comment]
I tried to change the commit email, but I made a mistake with git.
Let me challenge me again with this PR(https://github.com/loadimpact/k6/pull/1641).

**Pull request closed without merging**